backend/fms_core/import_tool/importers/experiment_run.py

```diff
@@ -0,0 +1,138 @@
+from pandas import pandas as pd
+from fms_core.models import ExperimentType, PropertyType
+from ._generic import GenericImporter
+from fms_core.import_tool.handlers import ExperimentRunHandler
+from .._utils import (data_row_ids_range,
+                      convert_property_value_to_str,
+                      blank_and_nan_to_none)
+
+
+class ExperimentRunImporter(GenericImporter):
+    base_errors = []
+    global_data = {'experiment_type': None, 'protocols_dict': {}, 'property_types_by_name': {}}
```

Comment:
global_data the name itself give me shivers ... This is unlikely to be the right way to deal with anything :) From what i see most of this can be made local and passed around as params.

Reply:
I can change the name (such as preloaded_data or so) but I want it to be a class variable to illustre the notion of it being related to the template, not to a specific row.